source/mainnet/smart-contracts/guides/setup-contract.rst

```diff
@@ -6,6 +6,38 @@
 Setting up a smart contract project
 ===================================
 
+This guide documents two different options (`from a template` or `from scratch`) to create a new Concordium smart contract project.
+The `from a template` option provides you with some
```

Comment:
I think we need to list the minimum version of `cargo-concordium` that supports the "from template" option.